source/aws-bootstrap-kit/test/aws-organizations-stack.test.ts

```diff
@@ -250,6 +253,20 @@ test("should not create any zone if no domain is provided", () => {
   expect(awsOrganizationsStack).toCountResources("AWS::Route53::HostedZone",0);
 });
 
+test("should not create root zone if existing root zone id is provided", () => {
+  const awsOrganizationsStack = new AwsOrganizationsStack(
+    new Stack(),
+    "AWSOrganizationsStack",
+    {
+      ...awsOrganizationsStackProps,
+      rootHostedZoneDNSName: "yourdomain.com",
+      existingRootHostedZoneId: "existing-root-zone-id"
+    }
+  );
+
+  expect(awsOrganizationsStack).toCountResources("AWS::Route53::HostedZone", 3);
```

Comment:
can we explicitly check there's 0 root hosted zone ?

Reply:
sure will do